app/models/list.rb

```diff
@@ -34,7 +34,7 @@ class List < ApplicationRecord
   private
 
   def validate_account_lists_limit
-    errors.add(:base, I18n.t('lists.errors.limit')) if account.lists.count >= PER_ACCOUNT_LIMIT
+    errors.add(:base, I18n.t('lists.errors.limit')) if account.owned_lists.count >= PER_ACCOUNT_LIMIT
```

Comment:
Ah, darn - looks like I grabbed the wrong association here - https://github.com/mastodon/mastodon/pull/31869/files#diff-52b06c9d36e8ab2e7afe4f11f44ce8a76306adf2d69e6df46c43e07e605e494aL31 - sorry for that miss.

I'm trying to think of what coverage would have stopped that ... but that PR literally was adding the coverage so it's sort of not relevant. My own confusion here was missing the difference between `lists` (records the account has been added to) vs `owned_lists` (records the account has created). I'm not sure which is the more intuitive naming ... one idea would be to rename `lists` to `targeted_lists` or something, and NOT HAVE the base `lists` association at all, to force contemplation of which thing you are doing.